### PR TITLE
ROC-2903: Mapper from CCD case to CMC claim

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-2903

### Change description ###
This PR adds mapping of CCD case to CMC claim.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 @Builder
 public class CCDCase {
 
+    private Long id;
```
Comment (rkondratowicz) [inline ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java]: is there are reason for this to not be a string?

Comment (sabahirfan) [reply ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java]: any reason to make this String?

Comment (rkondratowicz) [reply ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java]: swagger spec for ccd says it's a string.

Comment (rkondratowicz) [reply ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java]: actually, that's fine. nevermind.

**Pull request merged**